### PR TITLE
hooks/pre-pkg/05-rename-python3-c-bindings.sh: introduce

### PR DESCRIPTION
**Pull request by maxice8**
@Cogitri

### CODE REVIEW

**Comment by Cogitri** [comment]
Seems to work for me

File: common/hooks/pre-pkg/05-rename-python3-c-bindings.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# This hook executes the following tasks:
+#	- renames cpython binding files to not include the arch-specific extension suffix
+
+hook() {
+	if [ ! -d ${PKGDESTDIR}/${py3_sitelib} ]; then
+		return 0
+	fi
+
+	find "${PKGDESTDIR}/${py3_sitelib}" -type f -executable -iname '_[A-Za-z]*.cpython*.so' \
```
Comment (jnbr) [inline common/hooks/pre-pkg/05-rename-python3-c-bindings.sh]: the regex doesn't work for e.g. borg, should probably be `[A-Za-z_]*.cpython*.so` ?

**Comment by maxice8** [comment]
On Thu, Jan 31, 2019 at 05:35:13AM -0800, Johannes wrote:
> jnbr commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,16 @@
> +# This hook executes the following tasks:
> +#	- renames cpython binding files to not include the arch-specific extension suffix
> +
> +hook() {
> +	if [ ! -d ${PKGDESTDIR}/${py3_sitelib} ]; then
> +		return 0
> +	fi
> +
> +	find "${PKGDESTDIR}/${py3_sitelib}" -type f -executable -iname '_[A-Za-z]*.cpython*.so' \
> 
> the regex doesn't work for e.g. borg, should probably be `[A-Za-z_]*.cpython*.so` ?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/8012#pullrequestreview-198592058

made the proposed change, please test again.

**Comment by jnbr** [comment]
numbers need to be allowed as well (e.g. python3-pycrptodome). Is there a downside of simply matching `cpython.*.so` ?

**Comment by maxice8** [comment]
On Thu, Jan 31, 2019 at 06:13:28AM -0800, Johannes wrote:
> numbers need to be allowed as well (e.g. python3-pycrptodome). Is there a downside of simply matching `cpython.*.so` ?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/8012#issuecomment-459358517

I'd rather match as strictly as possible to avoid possible mistakes.

can it be '.cpython.*.so' ?

**Pull request merged**